jitpack.yml

```diff
@@ -0,0 +1,5 @@
+jdk:
```

Comment:
I assume we should also add an action to publish the package automatically for each new version (e..g on a new tag event)

Reply:
Meaning... somethig like we create a package to access to the latest package as part of publish, or after publishing npm package?

Jetpack builds a package when it gets an initial request. So to build a package on the jitpack, we need to create a dummy project which runs gradle sync to kick the build on the jitpack project. This is not actually "publishment" like npm, but let jitpack build this repository.

```
	dependencies {
	        implementation 'com.github.appium:appium-espresso-driver:v3.1.0'
	}
```

Reply:
Looks like the GH market has https://github.com/marketplace/actions/autojitpack so probably we could use it

Reply:
I have configured https://docs.jitpack.io/building/#building-ahead-of-time , so not every tag, but every push to master will build snapshot.